actions/utils/poll.ts

```diff
@@ -5,6 +5,7 @@ import {
   PollParams,
   PollResult,
 } from "@cowprotocol/cow-sdk";
+import { type } from "os";
```

Comment:
Not sure if this is necessary?

Reply:
deleging! i think it was my IDE, not conscious. nice catch